lib/modules/userHighlight.js

```diff
@@ -256,8 +280,37 @@ module.go = () => {
 	if (module.options.highlightSelf.value && username) {
 		highlight(`author[href$="/${username}"]`, module.options.selfColor.value, module.options.selfColorHover.value);
 	}
+	if (module.options.highlightOpMentions.value && isPageType('comments')) {
+		highlightMentionedUsers();
+	}
 };
 
+function highlightMentionedUsers() {
+	const links = getLinksFromContainer('#siteTable .usertext-body');
+	const usernames = getUsernamesFromLinks(links);
+	usernames.forEach(user => {
+		highlight(`author[href$="/${user}" i]`, module.options.opMentionsColor.value, module.options.opMentionsHover.value);
+	});
+}
+
+function getLinksFromContainer(containerSelector: string): NodeList<HTMLElement> {
+	return document.querySelectorAll(`${containerSelector} a`);
+}
+
+function getUsernamesFromLinks(links: NodeList<HTMLElement>): Set<string> {
+	const usernames = new Set();
+	const { protocol, hostname } = window.location;
+	links.forEach(link => {
+		const { href } = ((link: any): HTMLAnchorElement);
+		const url = new URL(href, `${protocol}//${hostname}`);
```

Comment:
Any reason for doing this instead of retrieving `hostname` and `pathname` directly from `link`?